x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/t1_analyst/post_detections_user.sh

```diff
@@ -0,0 +1,6 @@
+USER=(${@:-./detections_user.json})
+
+curl -v -H 'Content-Type: application/json' -H 'kbn-xsrf: 123'\
+ -u ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD} \
+-XPOST ${ELASTICSEARCH_URL}/_security/user/t1_analyst \
```

Comment:
When running these I'm seeing `Note: Unnecessary use of -X or --request, POST is already inferred.`. Removing `-XPOST` resolves this note.

```suggestion
 ${ELASTICSEARCH_URL}/_security/user/t1_analyst \
```